src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,22 @@
 package mate.academy
 
-class LetterRemover {
-    // implement ...
+import java.util.*
```

Comment:
The import statement for `java.util.*` is unnecessary in this file as none of the classes from this package are being used. Consider removing it to clean up the code.